index.js

```diff
@@ -0,0 +1,81 @@
+let fs = require('fs')
```

Comment:
cuando las variable mutan(cambian su valor) se usa let, si no mutan se usa const
```suggestion
const fs = require('fs')
```